### DISCUSSION TITLE
Dahua IMOU camera line

### DISCUSSION THREAD

**Discussion by husnainali09** [blakeblackshear/frigate#3664] [Ask A Question] [answered]
Am wanting to put a few of these indoor dahua IMOU cameras.

How is the general experience with these. Mrs won't let me put larger turrent mounted models indoors. So was wondering if these smaller wireless ones will work well with frigate ?

**Comment by amaduain** [comment, marked as answer]
Hi @husnainali09 ,
I have several outdoor IMOU cameras (Bullet 2C,  Bullet 2, and Bullet) all are working fine with Frigate after some tweaking:

- H.265 --> This is the default video codec, which can give you some problems, fortunately, IMOU provides the Config tool (https://www.imoulife.com/uk/support/download/tools) and allow you to change the video code to H.264
- Audio --> The recording audio was missing, but after proper configuration is copied to the stored file (https://github.com/blakeblackshear/frigate/issues/2506)

To get the connections urls for the cameras you can use this  https://www.ispyconnect.com/camera/imou, it will give you the URL to use on the configuration file usually something like: rtsp://user:password@ipaddress:554/cam/realmonitor?channel=1&subtype=0

Hope it helps!

**Reply by luciandf** [reply]
> Hi @husnainali09 , I have several outdoor IMOU cameras (Bullet 2C, Bullet 2, and Bullet) all are working fine with Frigate after some tweaking:
> 
>     * H.265 --> This is the default video codec, which can give you some problems, fortunately, IMOU provides the Config tool (https://www.imoulife.com/uk/support/download/tools) and allow you to change the video code to H.264
> 
>     * Audio --> The recording audio was missing, but after proper configuration is copied to the stored file ([[Solved]: No Audio or no Eventvideos #2506](https://github.com/blakeblackshear/frigate/issues/2506))
> 
> 
> To get the connections urls for the cameras you can use this https://www.ispyconnect.com/camera/imou, it will give you the URL to use on the configuration file usually something like: rtsp://user:password@ipaddress:554/cam/realmonitor?channel=1&subtype=0
> 
> Hope it helps!

can you upload that tool somewhere? they don't have it anymore

**Comment by amaduain** [comment]
https://dahuawiki.com/ConfigTool

**Comment by poseidogithub** [comment]
thanks this thread is gold to me!

**Comment by Linkatplug** [comment]
Hoooo dawn big thx for gold information

**Comment by bugui2007** [comment]
Just another comment about this cameras.
even using ConfigTool and set FPS to 5 for low quality stream0 that I used for motion detection on frigate, IMOU cameras allways stream0 FPS 15/1 and stream1 at FPS 25/1.
I try several times and I cant changed. 
Same issue happens with H264 and H265.
Old 2020's IMOU firmware works fine, but adjust FPS is broken now.

**Reply by luciandf** [reply]
is there a way to get back to that firmware?

**Comment by alex-richter** [comment]
I've also just run into a problem connecting my Imou Bullet Lite 4MP cameras to frigate. To get the cameras working properly, in the admin interface of the camera, a bit rate of 1280 kb/s must be selected. Otherwise, the video feed works but the playback of recordings in frigate doesn't. Maybe this helps someone!
